src/ui/public/agg_response/tabify/tabify.js

```diff
@@ -43,31 +43,31 @@ export function tabifyAggResponse(aggs, esResponse, respOpts = {}) {
  * @returns {undefined}
  */
 function collectBucket(write, bucket, key, aggScale) {
-  const agg = write.aggStack.shift();
+  const column = write.aggStack.shift();
+  const agg = column.aggConfig;
   const aggInfo = agg.write(write.aggs);
   aggScale *= aggInfo.metricScale || 1;
 
   switch (agg.type.type) {
     case 'buckets':
       const buckets = new TabifyBuckets(bucket[agg.id], agg.params);
       if (buckets.length) {
-        const splitting = write.canSplit && agg.schema.name === 'split';
-        if (splitting) {
-          write.split(agg, buckets, function forEachBucket(subBucket, key) {
-            collectBucket(write, subBucket, agg.getKey(subBucket, key), aggScale);
-          });
-        } else {
-          buckets.forEach(function (subBucket, key) {
-            write.cell(agg, agg.getKey(subBucket, key), function () {
-              collectBucket(write, subBucket, agg.getKey(subBucket, key), aggScale);
-            }, subBucket.filters);
-          });
-        }
+        buckets.forEach(function (subBucket, key) {
+          const bucketValue = agg.getKey(subBucket, key);
```

Comment:
I think this part needs some explaining comments. I don't get what it does by reading that code.